src/main/java/org/motechproject/mots/web/UserController.java

```diff
@@ -203,4 +203,72 @@ public UserProfileDto getUserProfile(Principal principal) {
 
     return permissionMapper.toDtos(permissions);
   }
+
+  /**
+   * Get Role with given id.
+   * @param id id of Role to find
+   * @return Role with given id
+   */
+  @RequestMapping(value = "/role/{id}", method = RequestMethod.GET)
+  @ResponseStatus(HttpStatus.OK)
+  @ResponseBody
+  public RoleDto getRole(@PathVariable("id") UUID id) {
+
+    UserRole role = userService.getRole(id);
+    return roleMapper.toDto(role);
+  }
+
+  /**
+   * Finds roles matching all of the provided parameters.
```

Comment:
missing javadoc

Reply:
@jkondrat this is the javadoc, so what do you mean?